docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eu
+
+# If the first argument looks like a flag, assume we want to run changedetection
+if [ "${1:0:1}" = '-' ]; then
+    set -- python ./changedetection.py -d /datastore "$@"
+fi
+
+# If we're running as root, by default make sure process uid/gid
+# and datadir permissions are correct. This can be skipped by setting
+# KEEP_PRIVILEGES to something non-empty.
+if [ "$(id -u)" = '0' -a -n "${KEEP_PRIVILEGES:-}" ]; then
+    PUID=${PUID:-911}
+    PGID=${PGID:-911}
+
+    groupmod -o -g "$PGID" changedetection
+    usermod -o -u "$PUID" changedetection
+
+    # Look for files in datadir not owned by the correct user and chown them
+    find "/datastore" \! -user changedetection -exec chown changedetection '{}' +
```

Comment:
I'm wondering if there could be a case where it would NOT be `/datastore` ? and if we can auto-detect what that path would be?

Reply:
Also any reference to `/datastore` should be in a single var if possible `$DATSTORE_DIR` ?

Reply:
It definitely should be defined in a var, agreed.

Off the top of my head, I can't think of a reason to override the defaults for the `/datadir` inside the container since anything can be mounted there. Autodetection might add unnecessary complexity + some unforeseen quirks imho, so I would assume anyone actually needing this for some reason should be experienced enough to do the necessary adjustments by themselves. We could parse the CLI arguments of course, but I'm not sure what the benefit would be.